### ISSUE TITLE
Don't Kill the Fat One quest is bugged

### ISSUE THREAD

**Issue by netherspite** [ccshiro/cc-buglist#653]
First of all, out of 6 times I tried to get Unkor the Ruthless talking to me 5 times he just died and never became green.
![wowscrnshot_060615_214928](https://cloud.githubusercontent.com/assets/5777848/8021006/29ad7514-0c98-11e5-8f1d-94a606bea826.jpg)

One of these 6 times he became green but was stuck running in place.
![wowscrnshot_060615_213045](https://cloud.githubusercontent.com/assets/5777848/8020993/f7a018a6-0c97-11e5-9eeb-e96ca5c53925.jpg)
After I moved far enough away for him to become not visible and moved back to him, he appeared green and sitting, willing to accept the Q, for few seconds, but before I was close enough to talk he evaded and transformed back into red agressive mob.

**Comment by netherspite** [comment]
More information: I managed to find the conditions of him becoming green. He only becomes green if you beat him AFTER killing 10 of the other ogres (Boulderfist Invaders) and does that only ONCE. 
On the second attempt when you already have "Unkor Submits" quest credit completed, he won't become green and will just die.
Both of those conditions are wrong.
http://www.wowhead.com/quest=9889/dont-kill-the-fat-one - here is a comment from patch 2.0.6 stating that if he is beaten up BEFORE killing 10 of the other ogres he still should become green but will not accept the quest because you didn't complete it yet (and you still can finish the quest by killing all the required ogres and then talk to him if he didn't evade yet).
Also there is a 3 minute evade timer information for him (the time after which he becomes back aggressive). Right now he is evading too fast (a minute or so).

Also, second time I managed to get him green I did that close to his spawnpoint and this time he sat down correctly instead of running in place. However, he still didn't want to accept the quest (there was no question mark above him, even gray one).
![wowscrnshot_060615_222837](https://cloud.githubusercontent.com/assets/5777848/8021143/51c6335c-0c9c-11e5-851d-bfd2d7b8c811.jpg)
